packages/cicd/github-actions/src/section/license-section-generator.adapter.ts

```diff
@@ -10,21 +18,135 @@ export class LicenseSectionGenerator extends GitHubActionsSectionGeneratorAdapte
 
     generateSection(formatterAdapter: FormatterAdapter, manifest: GitHubAction | GitHubWorkflow, repository: Repository): Buffer {
         const currentYear = new Date().getFullYear();
-        const authorName = ('author' in manifest && manifest.author) ? manifest.author : 'Your Name';
+        const authorName = ('author' in manifest && manifest.author) ? manifest.author : repository.owner;
+
+        // Use license information from repository (fetched by GitHubRepositoryProvider)
+        let licenseInfo: LicenseInfo | null = repository.license ? {
+            name: repository.license.name,
+            spdxId: repository.license.spdxId,
+            url: repository.license.url
+        } : null;
+
+        // Fallback to reading license file directly if no license info from GitHub
+        if (!licenseInfo) {
+            licenseInfo = this.getLicenseFromFile();
+        }
```

Comment:
Fallback must implemented in packages/repository/github/src/github-repository.provider.ts